src/io/flutter/view/MultiIconSimpleColoredComponent.java

```diff
@@ -224,6 +217,20 @@ public void append(@NotNull final String fragment, @NotNull final SimpleTextAttr
     revalidateAndRepaint();
   }
 
+  /**
+   * Adds the icon at the beginning of the line unlike addIcon which adds the
+   * icon after the already appended text.
+   */
+  @Override
+  public void setIcon(@Nullable Icon icon) {
+    assert(myIcons.size() == 0);
+    if (icon == null) {
+      return;
+    }
+
+    myIcons.add(new PositionedIcon(icon,0));
```

Comment:
I'd do a format of this file before committing it.

Reply:
done